catroid/src/main/res/values/strings.xml

```diff
@@ -753,6 +753,7 @@
         For further details on the license see the link below.\n</string>
     <string name="dialog_about_license_link_text">Source code license</string>
     <string name="dialog_about_catrobat_link_text">About Catrobat</string>
+    <string name="dialog_about_catrobat_language_version" translatable="false">Catrobat language version: </string>
```

Comment:
```suggestion
    <string name="dialog_about_catrobat_language_version">Catrobat language version</string>
```
This text needs to be translatable. Also, it's better to put the ": " into the code, so that translators do not have to bother with it.